vector/v.random/testsuite/test_v_random.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+output = 'test01'

```

Comment:
Please, run [Black](https://black.readthedocs.io/en/stable/) on the file (`black test_v_random.py`). This will resolve most of the formatting issues for you.

While at it, run also Flake8 and Pylint and try to minimize the issues reported.